docs/reference/master-configuration-reference.md

```diff
@@ -113,3 +113,16 @@ Example:
 ```yaml
 labelWhiteList: "foo"
 ```
+
+### resyncPeriod
+
+The `resyncPeriod` option specifies the NFD API controller resync period
+and it's expressed in hours.
```

Comment:
Not necessarily hours :) Also, I'd suggest to give a bit more detailed description what this actually does

```suggestion
The `resyncPeriod` option specifies the NFD API controller resync period. The resync means nfd-master replaying all NodeFeature and NodeFeatureRule objects, thus effectively re-syncing all nodes in the cluster (i.e. ensuring labels, annotations, extended resources and taints are in place). Only has effect when the [NodeFeature](../usage/custom-resources.md#nodefeature) CRD API has been enabled with [`-enable-nodefeature-api`](master-commandline-reference.md#-enable-nodefeature-api).
```